### PR TITLE
2 packages from ocaml-community/cppo at 1.7.0

### PR DESCRIPTION
**Pull request by fpottier**
This pull-request concerns:
- `cppo.1.7.0`: Code preprocessor like cpp for OCaml
- `cppo_ocamlbuild.1.7.0`: Plugin to use cppo with ocamlbuild

---
* Homepage: https://github.com/ocaml-community/cppo
* Source repo: git+https://github.com/ocaml-community/cppo.git
* Bug tracker: https://github.com/ocaml-community/cppo/issues

---
:camel: Pull-request generated by opam-publish v2.3.0

### CODE REVIEW

File: packages/cppo/cppo.1.7.0/opam [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+homepage: "https://github.com/ocaml-community/cppo"
+doc: "https://ocaml-community.github.io/cppo"
+bug-reports: "https://github.com/ocaml-community/cppo/issues"
+depends: [
+  "ocaml" {>= "4.02.3"}
+  "dune" {>= "2.0"}
+  "base-unix"
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (hannesm) [inline packages/cppo/cppo.1.7.0/opam:27]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/cppo_ocamlbuild/cppo_ocamlbuild.1.7.0/opam [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+doc: "https://ocaml-community.github.io/cppo"
+bug-reports: "https://github.com/ocaml-community/cppo/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "2.0"}
+  "ocamlbuild"
+  "ocamlfind"
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (hannesm) [inline packages/cppo_ocamlbuild/cppo_ocamlbuild.1.7.0/opam:23]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/cppo/cppo.1.7.0/opam (hunk 1 above)
Comment (fpottier) [reply packages/cppo/cppo.1.7.0/opam:27]: Thanks for spotting these problems. Apparently a side effect of the fact that `cppo` was pinned (on my machine) when I ran `opam publish`.

Comment (hannesm) [reply packages/cppo/cppo.1.7.0/opam]: You're welcome. The `pinned` was an earlier flag used here -- but since some time, `dev` is the preferred flag.

**Comment by fpottier** [comment]
I have had a look at the CI failure messages, but I see nothing that is clearly related to `cppo`.

**Comment by fpottier** [comment]
Ping? Is this PR acceptable in spite of the apparent CI failures?

**Review by shonfeder — APPROVED** [review]
Only revdeps are failing and I spot checked several and see no indication that they stem from this update.

**Pull request merged**